src/pages/deriv-go/components/_dgo-hero.js

```diff
@@ -0,0 +1,315 @@
+        position: unset;
+        top: unset;
+        justify-content: start;
+    }
+    @media (max-width: 360px) {
+        flex-wrap: wrap;
+    }
+`
+
+const IosComingSoon = styled(LinkButton)`
+    color: var(--color-white);
+    width: auto;
+
+    @media ${device.mobileL} {
+        max-width: 100%;
+        white-space: nowrap;
+        margin-left: 0;
+    }
+
+    @media (max-width: 360px) {
+        white-space: nowrap;
+    }
+`
+
+const StyledContent = styled(Header)`
+    font-size: 6.4rem;
+
+    @media (max-width: 1322px) {
+        font-size: 4.2rem;
+    }
+`
+const NormalContent = styled(Header)`
+    font-size: 2.4rem;
+
+    @media (max-width: 1322px) {
+        font-size: 1.2rem;
+    }
+`
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 562px;
+
+    @media (max-width: 1320px) {
+        max-width: 450px;
+    }
+    @media ${device.laptop} {
+        max-width: 390px;
+    }
+    @media ${device.laptopM} {
+        max-width: 410px;
+    }
+    @media ${device.tabletL} {
+        width: 46%;
+        max-width: 400px;
+        top: 60px;
+    }
+    @media ${device.tablet} {
+        width: 100%;
+        top: 350px;
+    }
+    @media ${device.mobileL} {
+        top: 280px;
+        max-width: 328px;
+    }
+    @media ${device.mobileM} {
+        top: 230px;
+    }
+`
+
+const query = graphql`
+    query {
+        mobile_float: file(relativePath: { eq: "mobile_float.png" }) {
+            childImageSharp {
+                fluid(maxWidth: 1024, srcSetBreakpoints: [1440]) {
+                    ...GatsbyImageSharpFluid_withWebp_noBase64
+                    originalName
+                }
+            }
+        }
+    }
+`
+
+const DGoHero = ({
+    background_alt,
+    content,
+    description,
+    google_play,
+    image_name,
+    is_mobile,
+    ios_coming_soon,
+    Logo,
+}) => {
+    const data = useStaticQuery(query)
+    // const getRedirectLink = () => {
+    //     const path = image_name === 'dbot' ? '/bot' : '/'
+    //     return path
+    // }
+
+    // const BackgroundSVG = styled.img`
+    //     position: absolute;
+    //     top: 0;
+    //     right: 0;
+    //     height: 100%;
+
+    //     @media ${device.laptopM} {
+    //         width: 100%;
+    //         max-width: 1440px;
+    //         height: initial;
+    //     }
+    //     @media ${device.laptop} {
+    //         width: 100%;
+    //     }
+    //     @media ${device.tabletL} {
+    //         width: 100%;
+    //         max-width: 350px;
+    //     }
+    //     @media ${device.tablet} {
+    //         width: 100%;
+    //     }
+    //     @media ${device.tabletS} {
+    //         width: 80%;
+    //         max-width: 337px;
+    //     }
+    //     @media ${device.mobileL} {
+    //         max-width: 250px;
+    //         min-height: 244px;
+    //     }
+    //     @media ${device.mobileM} {
+    //         max-width: 205px;
+    //         min-height: 0;
+    //     }
+    // `
```

Comment:
Any reason why the code above is commented?

Reply:
hadn't done code clean-up yet